week-4/Homework/mandatory/3-water-bottle.js

```diff
@@ -20,21 +20,41 @@ You have to implement the missing features according to the specification.
 // Here is your starting point:
 let bottle = {
   volume: 0,
-  fillUp: function() {
-    // calling this function should pour your bottle full (volume = 100);
+  fillUp: function () {
+    return this.volume === 100;
   },
+
   pour: function () {
     // calling this function should increase your bottle volume by 10 unit;
+
+    if (this.volume <= 90) {
+      this.volume += 10;
+    } else {
+      this.volume === 100;
+    }
   },
-  drink: function() {
+  drink: function () {
     // calling this function should decrease your bottle volume by 10 unit;
+
+    if (this.volume >= 10) {
+      this.volume -= 10;
+    } else {
+      this.volume === 0;
+    }
   },
   isFull: function () {
     // this function should return true if your bottle is empty;
+    if (this.volume >= 100) {
+      return this.volume === 100;
+    }
   },
-  isEmpty: function() {
+  isEmpty: function () {
     // this function should return true if your bottle is full;
-  }
+
+    if (this.volume === 0) {
+      return this.volume === 0;
+    }
+  },

```

Comment:
isFull and isEmpty should return true or false in each case. Again, replace '===' for assignment statements.

Reply:
Not to sure about this one